test/features/filters_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "./test/test_helper"
+
+class Filters < ActionDispatch::IntegrationTest
+  def test_it_filters_by_status_in_columns
+    user = create(:person)
+    create(:application, {person_id: user.id})
+    create(:application, {status: "applied", company: 'quickleft', person_id: user.id})
+    page.set_rack_session(user_id: user.id)
+    visit dashboard_path
+
+    within(".applied") do
+      assert page.has_content? 'quickleft'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.